openmetadata-ui/src/main/resources/ui/src/assets/svg/explore-vertical-nav-icons/ic-platform-lineage.svg

```diff
@@ -0,0 +1,7 @@
+<svg width="15" height="17" viewBox="0 0 15 17" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Let's remove height & width from SVG file
Also do not stroke color here, instead keep it currentColor and assign it through CSS